package.json

```diff
@@ -44,8 +44,8 @@
     "zone.js": "^0.8.5"
   },
   "peerDependencies": {
-    "@angular/core": "^2.1.0 || ^4.0.0",
-    "@angular/forms": "^2.1.0 || ^4.0.0",
-    "@angular/common": "^2.1.0 || ^4.0.0"
+    "@angular/core": "^2.1.0 || >=4.0.0",
```

Comment:
This is an pretty optimistic approach, so this problem does not appear again with the next major upgrade.

Please let me know if you prefer something that is easier to control.

Reply:
Can confirm that it properly works with Angular 6.